apps/crn-server/src/controllers/research-outputs.ts

```diff
@@ -376,6 +377,7 @@ export interface ResearchOutputController {
   fetchById: (id: string) => Promise<ResearchOutputResponse>;
   create: (
     researchOutputRequest: ResearchOutputCreateData,
+    createOptions?: { publish: boolean },
```

Comment:
```suggestion
    createOptions?: { publish?: boolean },
```

Reply:
Hey, I think we don't need this `?` because are always sending `publish` to the controller here

https://github.com/yldio/asap-hub/blob/CRN-792/apps/crn-server/src/routes/research-outputs.route.ts#L76
 right?